src/lib/esnext.object.d.ts

```diff
@@ -0,0 +1,13 @@
+interface ObjectConstructor {
+    /**
+     * Returns an object from an array of properties key/values
+     * @param entries Array that contains the properties key and value.
+     */
+    fromEntries<T>(entries: ArrayLike<[string, T]> | Iterable<[string, T]>): Record<string, T>;
+
+    /**
+     * Returns an object from an array of properties key/values
+     * @param entries Array that contains the properties key and value.
+     */
+    fromEntries(entries: ArrayLike<[string, any]> | Iterable<[string, any]>): Record<string, any>;
```

Comment:
https://github.com/tc39/ecma262/pull/1274

`fromEntries` only accepts iterable object so probably we should remove `ArrayLike` here. The Firefox implementation also throws for array-like object.